app/ts/components/App.tsx

```diff
@@ -197,7 +197,7 @@ export function App() {
 				}
 				case 'popup_websiteIconChanged': return setTabConnection(parsed.data)
 				case 'popup_new_block_arrived': {
-					await sendPopupMessageToBackgroundPage({ method: 'popup_refreshHomeData' })
+					sendPopupMessageToBackgroundPage({ method: 'popup_refreshHomeData' })
```

Comment:
Every call to an async function from a synchronous function should wrap the async call in `catchAllErrorsAndCall` or some similar function with a form like:
```ts
async function handleAllAsyncExceptions(func: () => void): void {
	try {
		await func()
	} catch (error) {
		// TODO: handle all possible errors, don't let anything bubble
	}
}
```
```suggestion
					handleAllAsyncExceptions(() => sendPopupMessageToBackgroundPage({ method: 'popup_refreshHomeData' }))
```